src/views/table_element_footer.ts

```diff
@@ -34,6 +37,28 @@ export class TableElementFooterView<PropsType extends BaseNodeProps>
     this.contentDOM = document.createElement('div')
```

Comment:
not sure if related but I we usually call updateContents in initialise for many nodeviews. Maybe worth checking

Reply:
Actually, it doesn't sound related. Thanks!